	FROM pg_catalog.pg_namespace n 
	WHERE n.nspname !~ '^pg_' AND n.nspname <> 'information_schema';
2022-08-01 18:44:16.278 UTC [9660] ERROR:  schema "public" already exists
2022-08-01 18:44:16.278 UTC [9660] STATEMENT:  CREATE SCHEMA public; SET search_path TO public;
2022-08-01 18:44:16.281 UTC [9660] ERROR:  could not find pg_class tuple for index 2662 at character 14
2022-08-01 18:44:16.281 UTC [9660] STATEMENT:  CREATE TABLE testns.nummultirange_test (nmr int);
2022-08-01 18:44:16.281 UTC [9660] ERROR:  could not find pg_class tuple for index 2662
2022-08-01 18:44:16.281 UTC [9660] STATEMENT:  GRANT ALL ON SCHEMA testns TO regress_priv_user1;
2022-08-01 18:44:16.282 UTC [9660] ERROR:  could not find pg_class tuple for index 2662 at character 8
2022-08-01 18:44:16.282 UTC [9660] STATEMENT:  SELECT has_table_privilege('regress_priv_user1', 'testns.nummultirange_test', 'SELECT');
2022-08-01 18:44:16.282 UTC [9660] ERROR:  could not find pg_class tuple for index 2662 at character 8
2022-08-01 18:44:16.282 UTC [9660] STATEMENT:  SELECT has_table_privilege('regress_priv_user1', 'testns.nummultirange_test', 'INSERT');
2022-08-01 18:44:16.282 UTC [9660] ERROR:  could not find pg_class tuple for index 2662
2022-08-01 18:44:16.282 UTC [9660] STATEMENT:  ALTER DEFAULT PRIVILEGES IN SCHEMA testns,testns GRANT SELECT ON TABLES TO public,public;
2022-08-01 18:44:16.282 UTC [9660] ERROR:  could not find pg_class tuple for index 2662 at character 8
2022-08-01 18:44:16.282 UTC [9660] STATEMENT:  SELECT has_table_privilege('regress_priv_user1', 'testns.nummultirange_test', 'SELECT');
2022-08-01 18:44:16.282 UTC [9660] ERROR:  could not find pg_class tuple for index 2662 at character 8
2022-08-01 18:44:16.282 UTC [9660] STATEMENT:  SELECT has_table_privilege('regress_priv_user1', 'testns.nummultirange_test', 'INSERT');
2022-08-01 18:44:16.282 UTC [9660] ERROR:  could not find pg_class tuple for index 2662 at character 8
2022-08-01 18:44:16.282 UTC [9660] STATEMENT:  SELECT has_table_privilege('regress_priv_user1', 'testns.nummultirange_test', 'INSERT');
2022-08-01 18:44:16.282 UTC [9660] ERROR:  role "regress_priv_user1" does not exist
2022-08-01 18:44:16.282 UTC [9660] STATEMENT:  ALTER DEFAULT PRIVILEGES IN SCHEMA testns GRANT INSERT ON TABLES TO regress_priv_user1;
2022-08-01 18:44:16.282 UTC [9660] ERROR:  could not find pg_class tuple for index 2662
2022-08-01 18:44:16.282 UTC [9660] STATEMENT:  CREATE FUNCTION unbounded_syntax_test2b(unbounded int) RETURNS TABLE (a int, b int, c int)
	LANGUAGE SQL
	AS $$
	  SELECT sum(unique1) over (rows between unbounded preceding and unbounded following),
	         unique1, four
	  FROM tenk1 WHERE unique1 < 10;
	$$;
2022-08-01 18:44:16.282 UTC [9660] ERROR:  could not find pg_class tuple for index 2662 at character 8
2022-08-01 18:44:16.282 UTC [9660] STATEMENT:  SELECT has_table_privilege('regress_priv_user1', 'testns.nummultirange_test', 'SELECT');
2022-08-01 18:44:16.282 UTC [9660] ERROR:  could not find pg_class tuple for index 2662 at character 8
2022-08-01 18:44:16.282 UTC [9660] STATEMENT:  SELECT has_table_privilege('regress_priv_user1', 'testns.nummultirange_test', 'INSERT');
2022-08-01 18:44:16.283 UTC [9660] ERROR:  could not find pg_class tuple for index 2662
2022-08-01 18:44:16.283 UTC [9660] STATEMENT:  ALTER DEFAULT PRIVILEGES GRANT USAGE ON SCHEMAS TO regress_priv_user2;
2022-08-01 18:44:16.283 UTC [9660] ERROR:  current transaction is aborted, commands ignored until end of transaction block
2022-08-01 18:44:16.283 UTC [9660] STATEMENT:  CREATE AGGREGATE testns.agg1(int) (sfunc = int4pl, stype = int4);
2022-08-01 18:44:16.283 UTC [9660] ERROR:  current transaction is aborted, commands ignored until end of transaction block
2022-08-01 18:44:16.283 UTC [9660] STATEMENT:  CREATE PROCEDURE testns.bar() AS 'select 1' LANGUAGE sql;
2022-08-01 18:44:16.283 UTC [9660] ERROR:  current transaction is aborted, commands ignored until end of transaction block
2022-08-01 18:44:16.283 UTC [9660] STATEMENT:  SELECT has_function_privilege('regress_priv_user2', 'testns.foo()', 'EXECUTE');
2022-08-01 18:44:16.283 UTC [9660] ERROR:  role "regress_priv_user1" does not exist
2022-08-01 18:44:16.283 UTC [9660] STATEMENT:  SET ROLE regress_priv_user1;
2022-08-01 18:44:16.283 UTC [9660] ERROR:  could not find pg_class tuple for index 2662
2022-08-01 18:44:16.283 UTC [9660] STATEMENT:  DROP FUNCTION testns.foo();
2022-08-01 18:44:16.283 UTC [9660] ERROR:  could not find pg_class tuple for index 2662 at character 8
2022-08-01 18:44:16.283 UTC [9660] STATEMENT:  SELECT has_function_privilege('regress_priv_user2', 'testns.bar()', 'EXECUTE');
2022-08-01 18:44:16.283 UTC [9660] ERROR:  could not find pg_class tuple for index 2662
2022-08-01 18:44:16.283 UTC [9660] STATEMENT:  DROP FUNCTION testns.foo();
2022-08-01 18:44:16.283 UTC [9660] ERROR:  could not find pg_class tuple for index 2662
2022-08-01 18:44:16.283 UTC [9660] STATEMENT:  DROP PROCEDURE testns.bar();
2022-08-01 18:44:16.283 UTC [9660] ERROR:  could not find pg_class tuple for index 2662
2022-08-01 18:44:16.283 UTC [9660] STATEMENT:  CREATE DOMAIN testns.nummultirange_test_btree AS int;
2022-08-01 18:44:16.283 UTC [9660] ERROR:  could not find pg_class tuple for index 2662
2022-08-01 18:44:16.283 UTC [9660] STATEMENT:  DROP PROCEDURE testns.bar();
2022-08-01 18:44:16.284 UTC [9660] ERROR:  could not find pg_class tuple for index 2662
2022-08-01 18:44:16.284 UTC [9660] STATEMENT:  ALTER DEFAULT PRIVILEGES IN SCHEMA testns GRANT USAGE ON TYPES to public;
2022-08-01 18:44:16.284 UTC [9660] ERROR:  could not find pg_class tuple for index 2662 at character 8
2022-08-01 18:44:16.284 UTC [9660] STATEMENT:  SELECT has_type_privilege('regress_priv_user2', 'testns.nummultirange_test_btree', 'USAGE');
2022-08-01 18:44:16.284 UTC [9660] ERROR:  could not find pg_class tuple for index 2662 at character 10
2022-08-01 18:44:16.284 UTC [9660] STATEMENT:  SELECT i,SUM(v::interval) OVER (ORDER BY i ROWS BETWEEN CURRENT ROW AND UNBOUNDED FOLLOWING)
	  FROM (VALUES(1,'1 sec'),(2,'2 sec'),(3,NULL),(4,NULL)) t(i,v);
2022-08-01 18:44:16.284 UTC [9660] ERROR:  could not find pg_class tuple for index 2662 at character 24
2022-08-01 18:44:16.284 UTC [9660] STATEMENT:  SELECT count(*)
	  FROM pg_default_acl d LEFT JOIN pg_namespace n ON defaclnamespace = n.oid
	  WHERE nspname = 'testns';
2022-08-01 18:44:16.284 UTC [9660] ERROR:  could not find pg_class tuple for index 2662
2022-08-01 18:44:16.284 UTC [9660] STATEMENT:  DROP SCHEMA testns CASCADE;
2022-08-01 18:44:16.284 UTC [9660] ERROR:  could not find pg_class tuple for index 2662
2022-08-01 18:44:16.284 UTC [9660] STATEMENT:  DROP SCHEMA testns2 CASCADE;
2022-08-01 18:44:16.284 UTC [9660] ERROR:  could not find pg_class tuple for index 2662
2022-08-01 18:44:16.284 UTC [9660] STATEMENT:  DROP SCHEMA testns2 CASCADE;
2022-08-01 18:44:16.284 UTC [9660] ERROR:  could not find pg_class tuple for index 2662
2022-08-01 18:44:16.284 UTC [9660] STATEMENT:  DROP SCHEMA testns3 CASCADE;
2022-08-01 18:44:16.284 UTC [9660] ERROR:  could not find pg_class tuple for index 2662 at character 8
2022-08-01 18:44:16.284 UTC [9660] STATEMENT:  SELECT VAR_SAMP(n::int) OVER (ORDER BY i ROWS BETWEEN CURRENT ROW AND UNBOUNDED FOLLOWING)
	  FROM (VALUES(1,600),(2,470),(3,170),(4,430),(5,300)) r(i,n);
2022-08-01 18:44:16.284 UTC [9660] ERROR:  could not find pg_class tuple for index 2662 at character 8
2022-08-01 18:44:16.284 UTC [9660] STATEMENT:  SELECT VARIANCE(n::int) OVER (ORDER BY i ROWS BETWEEN CURRENT ROW AND UNBOUNDED FOLLOWING)
	  FROM (VALUES(1,600),(2,470),(3,170),(4,430),(5,300)) r(i,n);
2022-08-01 18:44:16.285 UTC [9660] ERROR:  could not find pg_class tuple for index 2662
2022-08-01 18:44:16.285 UTC [9660] STATEMENT:  SELECT STDDEV_SAMP(n::smallint) OVER (ORDER BY i ROWS BETWEEN CURRENT ROW AND UNBOUNDED FOLLOWING)
	  FROM (VALUES(1,NULL),(2,600),(3,470),(4,170),(5,430),(6,300)) r(i,n);
2022-08-01 18:44:16.285 UTC [9660] ERROR:  could not find pg_class tuple for index 2662 at character 8
2022-08-01 18:44:16.285 UTC [9660] STATEMENT:  SELECT STDDEV(n::int) OVER (ORDER BY i ROWS BETWEEN CURRENT ROW AND UNBOUNDED FOLLOWING)
	  FROM (VALUES(0,NULL),(1,600),(2,470),(3,170),(4,430),(5,300)) r(i,n);
2022-08-01 18:44:16.285 UTC [9660] ERROR:  could not find pg_class tuple for index 2662 at character 16
2022-08-01 18:44:16.285 UTC [9660] STATEMENT:  SELECT to_char(SUM(n::float8) OVER (ORDER BY i ROWS BETWEEN CURRENT ROW AND 1 FOLLOWING),'999999999999999999999D9')
	  FROM (VALUES(1,1e20),(2,1)) n(i,n);
2022-08-01 18:44:16.285 UTC [9660] WARNING:  there is no transaction in progress
2022-08-01 18:44:16.285 UTC [9660] ERROR:  could not find pg_class tuple for index 2662
2022-08-01 18:44:16.285 UTC [9660] STATEMENT:  DISCARD ALL;
2022-08-01 18:44:16.285 UTC [9660] ERROR:  could not find pg_class tuple for index 2662 at character 24
2022-08-01 18:44:16.285 UTC [9660] STATEMENT:  SELECT n.nspname 
	FROM pg_catalog.pg_namespace n 
	WHERE n.nspname !~ '^pg_' AND n.nspname <> 'information_schema';
2022-08-01 18:44:16.285 UTC [9660] ERROR:  schema "public" already exists
2022-08-01 18:44:16.285 UTC [9660] STATEMENT:  CREATE SCHEMA public; SET search_path TO public;
2022-08-01 18:44:16.936 UTC [11020] PANIC:  could not open critical system index 2662
2022-08-01 18:44:16.939 UTC [9499] LOG:  server process (PID 11020) was terminated by signal 6: Aborted
2022-08-01 18:44:16.939 UTC [9499] LOG:  terminating any other active server processes
2022-08-01 18:44:17.001 UTC [9499] LOG:  all server processes terminated; reinitializing
2022-08-01 18:44:17.014 UTC [11090] LOG:  database system was interrupted; last known up at 2022-08-01 18:43:55 UTC
2022-08-01 18:44:17.030 UTC [11090] LOG:  database system was not properly shut down; automatic recovery in progress
2022-08-01 18:44:17.031 UTC [11090] LOG:  redo starts at 0/167D7A8
2022-08-01 18:44:17.277 UTC [11090] LOG:  invalid record length at 0/1D51750: wanted 24, got 0
2022-08-01 18:44:17.277 UTC [11090] LOG:  redo done at 0/1D51718 system usage: CPU: user: 0.16 s, system: 0.01 s, elapsed: 0.24 s
2022-08-01 18:44:17.352 UTC [9499] LOG:  database system is ready to accept connections
2022-08-01 18:44:18.200 UTC [11172] WARNING:  there is no transaction in progress
2022-08-01 18:44:18.257 UTC [11176] ERROR:  jsonpath member accessor can only be applied to an object
